include/openenclave/internal/datetime.h

```diff
@@ -52,6 +52,16 @@ int32_t oe_datetime_compare(
     const oe_datetime_t* date1,
     const oe_datetime_t* date2);
 
+/**
+ * Return the current system time in GMT time.
+ */
+oe_result_t oe_datetime_now(oe_datetime_t* value);
+
+/**
+ * Log date at the INFO level.
+ */
```

Comment:
I suggest changing this to VERBOSE level

Reply:
Done.